x-pack/solutions/search/plugins/search_getting_started/public/components/tutorials/console_tutorials_group.tsx

```diff
@@ -0,0 +1,180 @@
+  title: string;
+  dataTestSubj: string;
+  description: string;
+  request: string;
+  image: string;
+  buttonRef: React.RefObject<HTMLButtonElement>;
+}
+
+export const ConsoleTutorialsGroup = () => {
+  const { application, console: consolePlugin, share } = useKibana().services;
+  const tutorials: TutorialMetadata[] = [
+    {
+      title: i18n.translate('xpack.searchGettingStarted.consoleTutorials.basicsTitle', {
+        defaultMessage: 'Search basics',
+      }),
+      dataTestSubj: 'console_tutorials_search_basics',
+      description: i18n.translate('xpack.searchGettingStarted.consoleTutorials.basicsDescription', {
+        defaultMessage: 'Learn how to create an index, add documents, and basic search techniques.',
+      }),
+      request: consoleTutorials.basics,
+      image: searchWindowIllustration,
+      buttonRef: useRef<HTMLButtonElement>(null),
+    },
+    {
+      title: i18n.translate('xpack.searchGettingStarted.consoleTutorials.semanticTitle', {
+        defaultMessage: 'Intro to semantic search',
+      }),
+      dataTestSubj: 'console_tutorials_semantic_search',
+      description: i18n.translate(
+        'xpack.searchGettingStarted.consoleTutorials.semanticDescription',
+        {
+          defaultMessage:
+            'Learn semantic search techniques to understand intent and deliver more accurate, relevant results.',
+        }
+      ),
+      request: consoleTutorials.semanticSearch,
+      image: searchResultsIllustration,
+      buttonRef: useRef<HTMLButtonElement>(null),
+    },
+    {
+      title: i18n.translate('xpack.searchGettingStarted.consoleTutorials.esqlTitle', {
+        defaultMessage: 'ES|QL fundamentals',
+      }),
+      dataTestSubj: 'console_tutorials_esql',
+      description: i18n.translate('xpack.searchGettingStarted.consoleTutorials.esqlDescription', {
+        defaultMessage:
+          "Learn how to use Elastic's piped query language to simplify data investigations.",
+      }),
+      request: consoleTutorials.esql,
+      image: searchObserveIllustration,
+      buttonRef: useRef<HTMLButtonElement>(null),
+    },
+    // TODO:  uncomment below lines when we are ready to show TSDS tutorial. review https://github.com/elastic/kibana/pull/237384#issuecomment-3411670210
+    // {
+    //   title: i18n.translate('xpack.searchGettingStarted.consoleTutorials.tsdsTitle', {
+    //     defaultMessage: 'Time series data streams',
+    //   }),
+    //   dataTestSubj: 'console_tutorials_tsds',
+    //   description: i18n.translate('xpack.searchHomepage.consoleTutorials.tsdsDescription', {
+    //     defaultMessage:
+    //       'Learn how to use a time series data stream (TSDS) to store timestamped metrics data.',
+    //   }),
+    //   request: consoleTutorials.timeSeriesDataStreams,
+    //   image: null,
+    //   buttonRef: useRef<HTMLButtonElement>(null),
+    // },
+  ];
+
+  return (
+    <EuiFlexGroup gutterSize="l" direction={'column'} justifyContent="spaceBetween">
```

Comment:
nit: if we're passing a constant string IMO `direction="column"` is more readable.